modules/consensus/synchronize_test.go

```diff
@@ -1161,18 +1161,14 @@ func TestIntegrationBroadcastRelayHeader(t *testing.T) {
 	cst1.cs.gateway.Broadcast("RelayHeader", validBlock.Header(), cst1.cs.gateway.Peers())
 	select {
 	case <-mg.broadcastCalled:
-		// Broadcast is called twice, once to broadcast blocks to peers <= v0.5.1
-		// and once to broadcast block headers to peers > v0.5.1.
-		<-mg.broadcastCalled
 	case <-time.After(500 * time.Millisecond):
 		t.Fatal("RelayHeader didn't broadcast a valid block header")
 	}
 }
 
 // TestIntegrationRelaySynchronize tests that blocks are relayed as they are
-// accepted and that peers stay synchronized. This test is header/block
-// broadcast agnostic. When build.Version <= 0.5.1 block relaying will be
-// tested. When build.Version > 0.5.1 header relaying will be tested.
+// accepted and that peers stay synchronized. This test is heade
+// broadcast agnostic.
```

Comment:
looks like the last sentence got mangled a bit. It can probably just be removed.